docs/Advanced-Usage/Chip-Communication.rst

```diff
@@ -0,0 +1,76 @@
+.. _chip-communication:
+
+Communicating with the Chip/DUT
+===============================
+
+What good is a chip if it can't communicate with the outside world? Chipyard designs communicate to the outside world in
+one of two ways:
+
+* using the Front-End Server (FESVR)
```

Comment:
This is wrong. Both DTM and TSI use FESVR, and the JTAG debug interface uses GDB.